UserInterface.adoc

```diff
@@ -1,44 +1,92 @@
 virtual keyboard.
 In the middle of all of those is the content view which you can expect to change
 when navigating around Zyn-Fusion
 
-.Status Panel
+==== Status Panel
+
 The status panel is where you'll find a few global parameters, the file menu,
 automation-learn/fine-tuning mode selectors, the panic button, master VU meter,
-and the information tray.
+and the information tray. It is always situated at the top of Zyn's UI.
+We will be reading the status panel right-to-left, since you'll most often be using
+the righthand side.
 
 Panic button::
     When getting started you'll want to be familiar with the panic button as
     it will instantly silence the audio produced by Zyn, no matter what state
     the synth is in. It's a safety device in case the sounds go out of control.
+
 Master VU meter::
     This meter will display the current audio output levels. It can be useful for a few things:
     * It'll let you decide on choosing an optimal sound level for your patch.
     * It might be helpful to find out which instance of Zyn-Fusion is currently making sounds, if you're working with a few of them at once.
     * It'll help you make sure your MIDI keyboard is connected properly.
+
+VOLUME slider::
+    This slider is used to change the global volume of ZynAddSubFX.
+
+// TODO: When this changes, use: Allows shifting the input note up/down by a specified amount.
+KEY SHIFT slider::
+    Allows shifting the relation between which note would be played originally and which is played now.
+    64 means no keyshifting.
+    Setting the key shift below 64 shifts the keys down by the absolute difference between 64 and the key shift value.
+    Setting it above 64 does the same, except that it shifts the keys up.
+    For example, 63 shift all the keys down by 1, and 65 shifts them up by 1.
+
 Fine indicator::
     This indicator shows when parameters are being finely adjusted. Fine adjust
     is trigged by holding *SHIFT* and it can be toggled by clicking on the
     indicator.
-Learn indicator::
-    This indicator shows when automation learning is in progress.
+
+// TODO: More info about this in the manual?
+BPM panel::
+    This panel allows setting the BPM used by the project. This BPM is used
+    by LFOs whose frequency is defined with a numerator and denominator.
+
 Information Tray::
     The information tray will show tooltips about Zyn's parameters, current
-    parameter values, as well as other status information. image:imgs/info-tray.png[]
+    parameter values, as well as other status information. Image below.
 
-////
-.Screenshot todo
-NOTE: There should be a screenshot for the 'information tray'
+// TODO: NRPN is not a thing anymore, we now what the BPM textbox
+image:imgs/info-tray.png[]
+
+// TODO: More info here
+MIDI Learn indicator::
+    This indicator shows when MIDI contoller learning is in progress.
+
+// TODO: More info here
+Learn indicator::
```

Comment:
I actually am a bit worried about this documentation style. It's something that I've done in the past and sort of burnt out of documentation work. Enumerating everything in front of the user tends to make hard to read documentation and with something at the scale of zynaddsubfx it makes it an impossible task for you as the documentor. I'd drop some of these elements if you don't have something to tell the user about in a narrative sense.